lib/alarmist/definition.ex

```diff
@@ -119,6 +121,10 @@ defmodule Alarmist.Definition do
     end
 
     quote do
+      def __get_alarm_def() do
```

Comment:
Maybe this shouldn't be a hidden function? Thoughts?
```suggestion
      def definition() do
```

Reply:
I could see some utility from exposing this.

Reply:
I like the idea of keeping it private for now. I was going to suggest adding a public method on `Alarmist.Compiler` to get it, but it seems better to pretty-print the AST at runtime. That would also allow non-defalarm created alarms. Of course, that's work, there may be an issue, and saving the string to enable the feature does the job. We can some experience with the feature and perhaps that will better inform my opinion, at least. I agree that this is useful.